fixtures/Dockerfile

```diff
@@ -6,14 +6,12 @@ COPY fixtures/package.json package.json
 COPY fixtures/package-lock.json package-lock.json
 RUN npm ci --ignore-scripts
 
-FROM python:3-alpine3.17
+FROM python:3-alpine3.19
```

Comment:
You might want a specific python version here to avoid unexpected upgrades and breakage.

Reply:
Yeah, fair, I've pinned to 3.13